### PR TITLE
Add-912_Sort_an_Array

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 912_Sort_an_Array.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#O(n) T(nlogn)
```
Comment (ryujumoto) [inline 912_Sort_an_Array.py]: #time: O()
#space: O()

File: 912_Sort_an_Array.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+from typing import List
+
+
+class Solution:
+    def sortArray(self, nums: List[int]) -> List[int]:
```
Comment (ryujumoto) [inline 912_Sort_an_Array.py]: There are many algorithms to sort array. Investigate some of them and implement them.

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I wrote time and space complexity and summarize the code.
Please Check

**Comment by ryujumoto** [comment]
Refer [this article](https://qiita.com/suecharo/items/30f5d817da4c948c3be6) and implement selection sort, insertion sort, and merge sort.

**Comment by Rwatana** [comment]
I did several way and submitted five files. Please check my code.

**Comment by ryujumoto** [comment]
There are a lot of things to be modified in the code, so I'll explain verbally next time.

**Comment by Rwatana** [comment]
I think I revised many points. Please check my code.

**Comment by ryujumoto** [comment]
Why did you not modify the code based on my instructions?
I pointed out
- using descriptive variable names and appropriate file names and method ones.
- not writing `object` in `class` declarations because it's a grammar of Python2
- writing what `n` means in time complexity and space one

, etc.

**Pull request closed without merging**